ansible_collections/arista/avd/roles/eos_designs/doc/connected-endpoints.md

```diff
@@ -264,8 +228,75 @@ port_profiles:
           short_esi: < 0000:0000:0000 | auto >
 ```
 
+### Network Ports
+
+```yaml
+# Network Ports | Optional
+# All switch_ports ranges are exanded into indidual port configurations.
+# Switches are matched with regex matching the full hostname.
```

Comment:
```suggestion
# Switches are matched with regular expression matching the full hostname.
# Regexes used are based on Python3 're' module.
# Please see https://docs.python.org/3/library/re.html for documentation of regex syntax.
```

Reply:
Changing to
`# Switches are matched with regular expressions, which must match the full hostname.`
ok?